node/pkg/p2p/p2p.go

```diff
@@ -138,49 +157,6 @@ func Run(obsvC chan *gossipv1.SignedObservation, obsvReqC chan *gossipv1.Observa
 			return fmt.Errorf("failed to subscribe topic: %w", err)
 		}
 
-		// Add our own bootstrap nodes
-
-		// Count number of successful connection attempts. If we fail to connect to any bootstrap peer, kill
-		// the service and have supervisor retry it.
-		successes := 0
-		// Are we a bootstrap node? If so, it's okay to not have any peers.
-		bootstrapNode := false
-
-		for _, addr := range strings.Split(bootstrapPeers, ",") {
-			if addr == "" {
-				continue
-			}
-			ma, err := multiaddr.NewMultiaddr(addr)
-			if err != nil {
-				logger.Error("Invalid bootstrap address", zap.String("peer", addr), zap.Error(err))
-				continue
-			}
-			pi, err := peer.AddrInfoFromP2pAddr(ma)
-			if err != nil {
-				logger.Error("Invalid bootstrap address", zap.String("peer", addr), zap.Error(err))
-				continue
-			}
-
-			if pi.ID == h.ID() {
-				logger.Info("We're a bootstrap node")
-				bootstrapNode = true
-				continue
-			}
-
-			if err = h.Connect(ctx, *pi); err != nil {
-				logger.Error("Failed to connect to bootstrap peer", zap.String("peer", addr), zap.Error(err))
-			} else {
-				successes += 1
-			}
```

Comment:
Why was this block removed? Is it not necessary? It doesn't appear to be in the code block above.

Does this mean the bootstrap nodes will try to connect to themselves? What happens if there is a failure to connect to a bootstrap peer?